cells/xschem/xschemrc

```diff
@@ -304,7 +315,41 @@ set toolbar_visible 1
 set tabbed_interface 1
 
 ###########################################################################
-#### SKYWATER PDK SPECIFIC VARIABLES
+#### CASE INSENSITIVE SYMBOL LOOKUP
+###########################################################################
+## this option might be useful on filesystems that are case insensitive and
+## on designs ported from windows where case of file names does not matter.
+## if this option is set symbol lookup will be case insensitive,
+## so a symbol reference 'AMPLI.SYM' will match with 'ampli.sym' or
+## Amply.sym on disk. File system must be case insensitive for this to work,
+## like FAT32 or NTFS.
+## Do not set this option if you don't know what you are doing.
+## Default: not enabled (0)
+# set case_insensitive 1
+
+###########################################################################
+#### HIDE GRAPHS IF NO SPICE DATA LOADED
+###########################################################################
+## if enabled graphs will be hidden if no data is loaded.
+## default: not enabled (0)
+# set hide_empty_graphs 0
```

Comment:
Do these need to be uncommented by the end user?

Reply:
This depends on user preferences. If you have:
`set hide_empty_graphs 1`
If no data is loaded the whole graph will disappear from the schematic.
If you have:
`set hide_empty_graphs 0`
Empty graphs (grid, axes) will remain visible even if no spice simulation data is loaded.
This behavior can be switched from the GUI (Simulation menu)

Reply:
ok, I'll leave that commented out to leave that as a user option.

Reply:
These lines are also in the SKY130 xschemrc so I added them here also to reduce the diff between them.

Reply:
Cool. Thanks for the explanation!